dataduct/steps/executors/custom_check.py

```diff
@@ -0,0 +1,21 @@
+import pandas.io.sql as pdsql
```

Comment:
Would it be worth naming this file to `redshift_checks.py` or something similar indicating it contains redshift connection and query methods?

Or should this be considered the one place in dataduct where recurly custom checks will be defined?

Reply:
Yeah, that's probably a better name.